src/web/src/components/SearchInput/DateSearchInput.tsx

```diff
@@ -0,0 +1,45 @@
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import useSearchValue from '../../hooks/use-search-value';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    input: {
+      fontSize: '1.6rem',
+      '&:hover': {
+        borderColor: theme.palette.primary.main,
+      },
+      '&:focus': {
+        borderColor: theme.palette.primary.main,
+      },
+      '& > *': {
+        fontSize: '1.6rem !important',
+        color: theme.palette.text.primary,
+      },
+      height: '55px',
+      backgroundColor: 'transparent',
+      paddingLeft: '10px',
+      paddingRight: '60px',
+      border: 'none',
+      outline: 'none',
+      color: theme.palette.text.primary,
+    },
+  })
+);
+
+const DateSearchInput = () => {
+  const classes = useStyles();
+  const { onTextChange } = useSearchValue();
+  const today = new Date().toISOString().split('T')[0];
+
+  return (
+    <>
+      <input
```

Comment:
Also, I'd like some feedback on this. What if we made the type of input here `date`?

Reply:
Hey thanks for the review - so for this PR I am only putting in the logic, but as mentioned I am going to be using the datepicker once we put in the advanced search dialogue:

https://v4.mui.com/components/pickers/#datepickers

This will make sure the format is not skewed up. Unfortunately if I do this in the searchbar, which is also an input I'll just end up trying to add a bunch of front end to make it look nice. The idea is that this date input will be seen in the advanced search modal that I am hoping to work on in addition to solving issue-2110.